packages/main/src/plugin/image-registry.ts

```
@@ -790,7 +807,111 @@ export class ImageRegistry {
+          status: 'normal',
+          updateAvailable: false,
+          message: 'Image is detected as local and cannot be updated from a remote registry',
+        };
+      } else {
+        // Actual error from registry (auth failure, network issues, etc.)
+        return {
+          status: 'error',
+          updateAvailable: false,
+          message: errorMessage,
+        };
+      }
```

Biome (2.3.13)

[error] This else clause can be omitted because previous branches break early.

Safe fix: Omit the else clause.


(lint/style/noUselessElse)